models/lever.yml

```diff
@@ -87,11 +101,17 @@ models:
 
   - name: lever__opportunity_enhanced
     description: Table of unique opportunities.
-    columns:
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - source_relation 
+            - opportunity_id
```

Comment:
This is a bit out of scope, but I believe we should add `application_id` to this uniqueness test. When running with some internal data I receive a test failure on this model because of individuals who applied to the same opportunity multiple times.

It looks like this is a known phenomena as well since [we explicitly call this behavior out](https://github.com/fivetran/dbt_lever_source/blob/6d605f9a0462c0e45307fa18dbcbb9c40b106ceb/models/stg_lever.yml#L5-L8) in the staging model for opportunity. It then must come to fruition once we do [this join](https://github.com/fivetran/dbt_lever/blob/850b6928bf054d186634ec6be3cca1279bbe2996/models/intermediate/int_lever__opportunity_application.sql#L27-L30).

Would you be able to look into this a bit more and either add the `application_id` to the uniqueness test if it isn't too out of scope. Let me know if this ends up breaching too far out of scope, but if we can it would be great to address this alongside this next release. Thanks!

Reply:
I opened [this issue ](https://github.com/fivetran/dbt_lever/issues/22) since I had to investigate if the model or the test had to be updated. I can look at it next if we're ok delaying release.

Reply:
Ah yes thank you, I remember you opening this issue and raising it to me previously. Apologies for missing it. Since this change (if it's just a test update) would likely be a patch update it should be fine to move this investigation to next week and get this change out to address the larger issue we're trying to address with this update.

Given the above, can we recategorize this issue as a Bug instead of a FR so we can make sure we prioritize it over other issues next sprint.